.chloggen/kafka-receiver-standardize-default-topic.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
```

Comment:
Should this be considered a breaking change? If a user is currently storing logs or metrics in the `otlp_spans` kafka topic and relying on the default `topic` value of the receiver's configuration, wouldn't this break their valid use-case?

(I agree with this change, just want to make sure it's communicated effectively to users 👍 )

Reply:
+1 I think we should make it a breaking change and describe mitigations

Reply:
Agreed. Swapped the `change_type` to `breaking`. Any other documentation I should come up with to address mitigations?

Reply:
There's no other documentation required that I'm aware of.

Reply:
I feel it would be good to add docs that would help users with upgrading, perhaps in the changelog entry's `subtext` section. Perhaps something along these lines?

```
subtext: |
  If you are using the Kafka receiver in a logs and/or a metrics pipeline
  and you are not customizing the name of the topic to read from with the `topic` property,
  the receiver will now read from `otlp_logs` or `otlp_metrics` topic instead of `otlp_spans` topic.
  To maintain previous behavior, set the `topic` property to `otlp_spans`.
```

Reply:
Great call. Sorry for the slow response. Been OOO with no laptop recently.

Reply:
Thank you! Looks good to me.